src/Storages/ObjectStorageQueue/ObjectStorageQueueMetadata.h

```diff
@@ -148,9 +150,17 @@ class ObjectStorageQueueMetadata
     /// Set local ref count for metadata.
     void setMetadataRefCount(std::atomic<size_t> & ref_count_) { chassert(!metadata_ref_count); metadata_ref_count = &ref_count_; }
 
+    void updateSettings(const SettingsChanges & changes);
+
+    std::pair<size_t, size_t> getCleanupInterval() const { return {cleanup_interval_min_ms, cleanup_interval_max_ms }; }
+
+    size_t usePersistentProcessingNode() const { return persistent_processing_node_ttl_seconds; }
```

Comment:
Looks like a typo: should be `return use_persistent_processing_nodes`, and boolean return value.